build/cloud/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM golang:1.12.1-alpine3.9 AS builder
+FROM golang:1.12-alpine3.9 AS builder
 
 COPY . /go/src/github.com/kubeedge/kubeedge
 
 RUN CGO_ENABLED=0 go build -v -o /usr/local/bin/cloudcore -ldflags="-w -s" \
 github.com/kubeedge/kubeedge/cloud/cmd/cloudcore
 
 
-FROM alpine:3.9
+FROM gcr.io/distroless/static
```

Comment:
I switched here to [distroless](https://github.com/GoogleContainerTools/distroless) which seems to be a nice fit here. My first tests where successfully. I can also move this change to another PR. I think we can also update other images to use distroless (has a smaller attack surface and requires less updates.